build/parser.js

```diff
@@ -506,12 +506,19 @@ class Parser {
       if (data.length) item.drops = data
     }
 
-    // Sort by drop rate
     if (item.drops) {
-      item.drops.sort((a, b) => b.chance - a.chance)
+      item.drops.sort(this.comparator)
     }
   }
 
+  // Compares drop locations for items lexicographically by chance + location + rotation + rarity
+  comparator (dropA, dropB) {
```

Comment:
you're gonna hate me, but could you call it a `dropComparator` instead, so it doesn't get confused with anything else we may want to compare in the future?